copier/cli.py

```diff
@@ -114,6 +123,7 @@ class CopierApp(cli.Application):
         """\
         copier [MAIN_SWITCHES] [copy] [SUB_SWITCHES] template_src destination_path
         copier [MAIN_SWITCHES] [update] [SUB_SWITCHES] [destination_path]
+        copier [MAIN_SWITCHES] check [SUB_SWITCHES] [destination_path]
```

Comment:
For consistency with the other subcommands:
```suggestion
        copier [MAIN_SWITCHES] [check] [SUB_SWITCHES] [destination_path]
```

Reply:
Aren't the []'s there to indicate that this is optional, that you can just call copier and it will guess what you want? This is not the case for the check command...?

Reply:
I think you're right. But `copier ...` without a subcommand does not automatically choose between generating a new project and updating it:

https://github.com/copier-org/copier/blob/91966047d9741099b4925612c5c0c3a0dbe2490c/copier/cli.py#L292-L296

Should the `copy` and `update` subcommands be documented with surrounding brackets, too, @yajo?